image/store.go

```diff
@@ -170,6 +171,9 @@ func (is *store) Search(term string) (ID, error) {
 
 	dgst, err := is.digestSet.Lookup(term)
 	if err != nil {
+		if strings.Contains(err.Error(), "digest not found") {
+			err = fmt.Errorf("Image %s could not found or does not exist.", term)
```

Comment:
Think this should be either "could not be found" (added 'be'), but perhaps it's better to use

    no such image: %s

As that's what we use in other locations